client/commonFramework/update-preview.js

```diff
@@ -13,7 +13,10 @@ window.common = (function(global) {
   window.__err = null;
   window.loopProtect.hit = function(line) {
     window.__err = new Error(
-      'Potential infinite loop at line ' + line
+      "Potential infinite loop at line " + line + ".\n\
+      To disable loop protection: add \"// noprotect\" to your code.\n\
```

Comment:
I wasn't able to get my linter to pick up the error, but was able to see the issue with the trailing `;` when pasting the JavaScript inside the template string to jshint.com :/

@BerkeleyTrue will look into getting that sorted out before committing anything else.